xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEBuffer.cpp

```diff
@@ -296,18 +293,10 @@ bool CActiveAEBufferPoolResample::ResampleBuffers(int64_t timestamp)
 
       if (in)
       {
-        if (!timestamp)
-        {
-          if (in->timestamp)
-            m_lastSamplePts = in->timestamp;
-          else
-            in->pkt_start_offset = 0;
-        }
```

Comment:
I removed the function value `int64_t timestamp`, as this was only used for visualizations and if unnecessary stream work is not used, it should be checked for" 0 ".